Bottom: 972340277be97383de01e22e46e4810d3e7cdba7
Top:    962ed03ec0d388b4d3b7339d629bee926d445873
Author: Jon Smirl (none) <jonsmirl@smirl2>
Date:   2012-03-19 23:01:40 -0400

Remove overlapping IO defs


---

diff --git a/arch/arm/mach-lpc31xx/generic.c b/arch/arm/mach-lpc31xx/generic.c
index 6729930..8ed8230 100644
--- a/arch/arm/mach-lpc31xx/generic.c
+++ b/arch/arm/mach-lpc31xx/generic.c
@@ -146,12 +146,14 @@ static struct map_desc lpc313x_io_desc[] __initdata = {
 		.length		= IO_APB4_SIZE,
 		.type		= MT_DEVICE
 	},
+#if 0
 	{
 		.virtual	= io_p2v(IO_DMA_REG_PHYS),
 		.pfn		= __phys_to_pfn(IO_DMA_REG_PHYS),
 		.length		= IO_DMA_REG_SIZE,
 		.type		= MT_DEVICE
 	},
+#endif
 	{
 		.virtual	= io_p2v(IO_MPMC_CFG_PHYS),
 		.pfn		= __phys_to_pfn(IO_MPMC_CFG_PHYS),
